libdl/Android.bp

```diff
@@ -71,6 +71,10 @@ cc_library {
     sanitize: {
         never: true,
     },
+
```

Comment:
libdl.so is a special case anyway: some of its functions are actually provided by the loader, by overriding symbols by name.